ccmlib/scylla_node.py

```diff
@@ -1204,19 +1204,18 @@ def _write_agent_log4j_properties(self, agent_dir):
         raise NotImplementedError('ScyllaNode._write_agent_log4j_properties')
 
     def _wait_no_pending_flushes(self, wait_timeout=60):
-        def no_pending_flushes():
-            stdout, _ = self.nodetool('cfstats', timeout=wait_timeout)
+        def no_pending_flushes() -> bool:
+            stdout = self.nodetool('cfstats', timeout=wait_timeout)[0]
             pending_flushes = False
             for line in stdout.splitlines():
                 line = line.strip()
                 if line.startswith('Pending flushes'):
-                    _, pending_flushes_str = line.split(':')
+                    pending_flushes_str = line.split(':', maxsplit=1)[1]
```

Comment:
Doesn't this maxsplit=1 change the behavior of this code?
I guess it will never matter because there is just one ":" anyway, but I'm not familiar enough with this code to say.